Document/0x08-Testing-Tools.md

```diff
@@ -4,11 +4,14 @@ To perform security testing different tools are available in order to be able to
 
 ### Mobile Application Security Testing Distributions
 
+- [Appie](https://manifestsecurity.com/appie/) - A portable software package for Android Pentesting and an awesome alternative to existing Virtual machines.
 - [Androl4b](https://github.com/sh4hin/Androl4b) - A Virtual Machine For Assessing Android applications, Reverse Engineering and Malware Analysis
 - [Android Tamer](https://androidtamer.com/) - Android Tamer is a Debian-based Virtual/Live Platform for Android Security professionals.
 - [AppUse](https://appsec-labs.com/AppUse/) - AppUse is a Virtual Machine developed by AppSec Labs.
+- [Mobisec](https://sourceforge.net/projects/mobisec/) - Mobile security testing live environment.

```

Comment:
Does this still work? seems 3 years old (can you verify it?)

Reply:
The latest update was made on Jun 2, 2018, and from what I can see it seems to be up to date.

Reply:
Latest update is from June 2018, and it looks like it is working as intended.

Reply:
Up to date as of June 2018.

Reply:
Working as of 31 Oct 2018